config/config.go

```diff
@@ -24,6 +24,7 @@ type Config struct {
 	DatasetAPIURL                string        `envconfig:"DATASET_API_URL"`
 	CodelistAPIURL               string        `envconfig:"CODELIST_API_URL"`
 	RecipeAPIURL                 string       ` envconfig:"RECIPE_API_URL"`
+	ImportAPIURL                 string       ` envconfig:"IMPORT_API_URL"`
```

Comment:
Would it make sense to add a README entry for this one.

Reply:
Now that you mention it, all of the environment variables need updating in the README